src/stk500v2.c

```diff
@@ -1254,10 +1262,10 @@ static int stk500v2_initialize(const PROGRAMMER *pgm, const AVRPART *p) {
     unsigned char vtarg_read;
     stk500v2_getparm(pgm, PARAM_VTARGET, &vtarg_read);
     if (PDATA(pgm)->vtarg_get)
-      msg_info("Target voltage value read as %.2fV\n", (vtarg_read / 10.0));
+      msg_info("Target voltage value read as %.2f V\n", (vtarg_read / 10.0));
     // Write target voltage value
     else {
-      msg_info("Changing target voltage from %.2f to %.2fV\n", (vtarg_read / 10.0), PDATA(pgm)->vtarg_data);
+      msg_info("Changing target voltage from %.2f V to %.2f V\n", (vtarg_read / 10.0), PDATA(pgm)->vtarg_data);
```

Comment:
`unsigned char vtarg_read;` should be initialised with 0 (same reason as below, needs to be done twice in this file).